test/mod/test_fmt_v2.py

```diff
@@ -80,6 +80,13 @@
                                 "filename": "empty.img"
                             }
                         },
+                        "var": {
+                            "type": "org.osbuild.loopback",
+                            "options": {
+                                "filename": "empty.img",
+                                "partscan": True,
+                            },
+                        },
```

Comment:
trying to understand this test.. does empty.img really exist or is it just mocked somehow? If it does exist I doubt this manifest would really work as you are mounting empty.img and empty.img partition 1 below.

Reply:
This test does not run a full pipeline, it just reads the json and checks that the data structures are correctly parsed/presented. I mostly added this so that we have something that uses "partition" because the "describe()" was missing it (and I did not noticed until I added the test data :)